packages/mui-material/src/Modal/ModalManager.test.ts

```diff
@@ -379,9 +379,9 @@ describe('ModalManager', () => {
       expect(container2.children.length).equal(numberOfChildren);
 
       modalManager.add(getDummyModal(), container2);
-      expect(container2.children[0]).toBeAriaHidden();
+      expect(container2.children[0]).toBeInaccessible();
       for (let i = 1; i < numberOfChildren; i += 1) {
-        expect(container2.children[i]).not.toBeAriaHidden();
+        expect(container2.children[i].getAttribute('aria-hidden')).to.equal(null);
```

Comment:
Unique case where we cannot use `.not.toBeInaccessible` assertion. Some elements in the forbidden list like `<template>, <script>, <style>, <noscript>` etc are implicitly inaccessible in the accessibility tree due to `display: 'none'` being applied implicitly. Other elements like `<map>`, `<col>`, `<colgroup>` etc are accessible.